.circleci/config.yml

```diff
@@ -108,3 +113,15 @@ workflows:
           on_tag: false
           requires:
             - test-policies
+      # Shared
+      - architect/push-to-app-catalog:
+          name: push-policies-shared-to-catalog
+          app_catalog: control-plane-test-catalog
```

Comment:
@MarcelMue is there a reason why we only deployed it on the test catalog ?

Reply:
No - I need to clean this up

Will fix `config` repo and this push throughout the day

Reply:
No worries :D 

Would it be enough https://github.com/giantswarm/config/pull/365  ?